typescript-selenium-webdriver/jest.config.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const rootDir = './';
+const currentDir = '<rootDir>/';
+
+module.exports = {
+    testRunner: 'jest-circus/runner',
```

Comment:
I think we can simplify this file a bit; the coverage and junit stuff will still be necessary for devops compat, but some of the rest can be dropped. I think this is minimal:

```typescript
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
module.exports = {
    preset: 'ts-jest',
    testEnvironment: 'node',

    collectCoverage: true,
    collectCoverageFrom: ['./**/*.ts', '!./**/*.test.ts'],
    coverageDirectory: './test-results/coverage',
    coverageReporters: ['json', 'lcov', 'text', 'cobertura'],
    reporters: [
        'default',
        [
            'jest-junit',
            {
                outputDirectory: '.',
                outputName: './test-results/junit.xml',
            },
        ],
    ],
};
```